src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/HomeOfficeChallenge.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities;
+
+import java.util.List;
+
+public class HomeOfficeChallenge {
+
+    private List<Person> applicants;
+    private String appealType;
+    private String appealTierType;
```

Comment:
Shouldn't this be enum?

Reply:
It is kept as String to reflect the home office model.